src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaTimelinePersistenceAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.TimelineEvent
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.timeline.service.TimelinePersistenceAdapter
+
+private val log = KotlinLogging.logger {}
+
+@Component
+class JpaTimelinePersistenceAdapter(
+  // private val timelineRepository: TimelineRepository,
+  // private val timelineMapper: TimelineEntityMapper,
+) : TimelinePersistenceAdapter {
+
+  @Transactional
+  override fun recordTimelineEvent(prisonNumber: String, event: TimelineEvent) {
+    log.info { "Recording TimelineEvent [$event] for prisoner [$prisonNumber]" }
+    // TODO RR-317
```

Comment:
I'll do all the JPA and database stuff in a separate PR (for which I've created a separate sub-task).